vmdb/db/migrate/20150521204814_add_container_relationship.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerRelationship < ActiveRecord::Migration
+  def change
+    add_column :containers, :container_definition_id, :bigint
```

Comment:
Once you'll be done with the other changes we'll probably have to discuss this part because we may want to drop other columns that are now useless.